infinispan/runtime/src/main/java/io/quarkiverse/langchain4j/infinispan/SchemaAndMarshallerProducer.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkiverse.langchain4j.infinispan;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.inject.Instance;
+import jakarta.enterprise.inject.Produces;
+import jakarta.inject.Inject;
+
+import org.infinispan.protostream.FileDescriptorSource;
+import org.infinispan.protostream.MessageMarshaller;
+
+import io.quarkiverse.langchain4j.infinispan.runtime.InfinispanEmbeddingStoreConfig;
+import io.quarkiverse.langchain4j.infinispan.runtime.LangchainItemMarshaller;
+
+@ApplicationScoped
+public class SchemaAndMarshallerProducer {
+
+    public static final String LANCHAIN_ITEM = "LangchainItem";
```

Comment:
There's still a missing G :)

Reply:
forgot to push that change :) crazy G xD